src/antelope/stores/feedback.ts

```diff
@@ -105,7 +105,15 @@ export const createTraceFunction = (store_name: string) => function(action: stri
 };
 
 
-export const isTracingAll = () => false;
+// only if we are NOT in production mode search in the url for the trace flag
+// to turn on the Antelope trace mode
+let trace = false;
+if (process.env.NODE_ENV !== 'production') {
+    const urlParams = new URLSearchParams(window.location.search);
+    trace = urlParams.get('trace') === 'true';
+}
+
+export const isTracingAll = () => trace;
```

Comment:
I added this new code to turn on this feature bu adding a simple param to the url.
But it is only available for non-production deployments.

however, somehow [this netlify link](https://deploy-preview-409--wallet-staging.netlify.app/) is set as such when it should not. Or am I wrong? 
@donnyquixotic,  can you check this? Thanks